rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -688,8 +696,16 @@ public ResponseEntity linkReleases(
                             // TODO: Add example for MAP value
                     }
             )
-            @RequestBody Object releasesInRequestBody
+            @RequestBody Object releasesInRequestBody,
+            @Parameter(description = "Comment message.")
+            @RequestParam(value = "comment", required = false) String comment
     ) throws URISyntaxException, TException {
+        User user = restControllerHelper.getSw360UserFromAuthentication();
+        Project updateProject = projectService.getProjectForUserById(id, user);
+        user.setCommentMadeDuringModerationRequest(comment);
+        if (!restControllerHelper.isWriteActionAllowed(updateProject, user) && comment == null) {
+            return new ResponseEntity<>(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?